modules/alma/includes/alma.user.inc

```diff
@@ -20,6 +20,10 @@ function alma_user_authenticate($uid, $pass) {
     if ($res) {
       $return['success'] = TRUE;
     }
+    elseif (NULL === $res) {
```

Comment:
Different comparison style across module.
We either keep the comparison style or change it everywhere.